### PR TITLE
Rhel8 alpha branch remove btrfs support in kickstart 3

### PR DESCRIPTION
**Pull request by rvykydal**

### CODE REVIEW

**Comment by rvykydal** [comment]
Related blivet PR (https://github.com/rhinstaller/anaconda/pull/1328/commits/0b28dcaf4f9264355cc0078d9e42e697829906d8):
https://github.com/storaged-project/blivet/pull/661

Related pykickstart PR:
https://github.com/clumens/pykickstart/pull/210

**Comment by rvykydal** [comment]
Obsoletes https://github.com/rhinstaller/anaconda/pull/1302

File: anaconda.spec.in [hunk 1]
```diff
@@ -37,7 +37,7 @@ Source0: %{name}-%{version}.tar.bz2
 %define mehver 0.23-1
 %define nmver 1.0
 %define partedver 1.8.1
-%define pykickstartver 3.10-1
+%define pykickstartver 3.12-1
```
[the remark this answers is not in the document]
Comment (rvykydal) [reply anaconda.spec.in]: Done

**Comment by Conan-Kudo** [comment]
😭

**Comment by rvykydal** [comment]
Rebased on added blivet 3 support. Which means we don't need to require blivet with deepcopy fix (patch dropped) but fix the deepcopy in anaconda storage code instead (added patch https://github.com/rhinstaller/anaconda/pull/1328/commits/17baa572e2e3e072a10392e8e395fb09be9deb27)

File: pyanaconda/ui/gui/spokes/installation_source.py [hunk 2]
```diff
@@ -412,16 +412,25 @@ def _method_changed(self):
...
+        # This is an ugly temporary fix, because we cannot use deepcopy since
+        # pykickstart 3. This entire module should be rewritten anyway.
+        old_method = self.data.method.method
+        old_partition = getattr(self.data.method, "partition", None)
+        old_dir = getattr(self.data.method, "dir", None)
+        old_url = getattr(self.data.method, "url", None)
+        old_server = getattr(self.data.method, "server", None)
+        old_opts = getattr(self.data.method, "opts", None)
+        old_mirrorlist = getattr(self.data.method, "mirrorlist", None)
+        old_metalink = getattr(self.data.method, "metalink", None)
```
Comment (jkonecny12) [inline pyanaconda/ui/gui/spokes/installation_source.py]: Beware on this. It will be hard to fix later in the product life.

**Comment by rvykydal** [comment]
jenkins, test this please

**Comment by rvykydal** [comment]
jenkins, test this please

**Pull request merged**